packages/search-spa/set-env.ts

```diff
@@ -0,0 +1,21 @@
+const fs = require('fs');
+
+const writeFile = fs.writeFile;
+
+const targetPath = './src/environments/environment.ts';
+
+require('dotenv').config();
+
+const envConfigFile = `export const environment = {
+   API_URL: '${process.env.API_URL}',
+   production: true,
```

Comment:
The production build should be configurable from the `.env`

```suggestion
   production: process.env.NODE_ENV === 'production',
```